### ISSUE TITLE
App won't open at all...

### ISSUE THREAD

**Issue by nikhiljha** [NemProject/NEMAndroidApp#244]
Relevant LogCat Lines
```
08-22 08:52:43.337 E/ACRA    (30171): ACRA caught a RuntimeException for org.nem.nac
08-22 08:52:43.337 E/ACRA    (30171): java.lang.RuntimeException: Unable to resume activity {org.nem.nac/org.nem.nac.ui.activities.SplashActivity}: java.lang.RuntimeException: org.nem.nac.ui.activities.SplashActivity@7923bd2 must implement OnFragmentInteractionListener
08-22 08:52:43.337 E/ACRA    (30171): Caused by: java.lang.RuntimeException: org.nem.nac.ui.activities.SplashActivity@7923bd2 must implement OnFragmentInteractionListener
08-22 08:52:43.337 E/ACRA    (30171):  at org.nem.nac.ui.dialogs.MainNetStartupWarningDialogFragment.onAttach(MainNetStartupWarningDialogFragment.java:83)
```

I think you can fix this bug by
- Setting compile target to the latest android, but leave that other target (compatibility?) at 4.4.4.
- Implementing the OnFragmentInteractionListener method in that specific dialog, as you can see in logcat.

**Comment by nikhiljha** [comment]
Confirmed same bug on my s3 with Android 6.

Looks like a 6+ bug.

Easy fix though. I detailed how to fix it already.

**Comment by am4n0w4r** [comment]
Fixed in 1.0.60;
Very strange that it wasn't crashing on all other systems as it should so I haven't noticed it..

**Comment by nikhiljha** [comment]
Tested and fixed on N and M.
